okhttp-hpacktests/pom.xml

```diff
@@ -8,7 +8,7 @@
   <parent>
     <groupId>com.squareup.okhttp3</groupId>
     <artifactId>parent</artifactId>
-    <version>3.3.0-SNAPSHOT</version>
```

Comment:
This was what made us vulnerable to the regression. We weren’t running HPACK tests against the current code; we were running them against a known working version.